pkgs/applications/graphics/pureref/default.nix

```
@@ -1,14 +1,17 @@
-{ lib, appimageTools, requireFile }:
+{ lib, appimageTools, runCommand, curl, gnugrep, cacert }:
 
 appimageTools.wrapType1 rec {
   pname = "pureref";
   version = "1.11.1";
 
-  src = requireFile {
-    name = "PureRef-${version}_x64.Appimage";
-    sha256 = "05naywdgykqrsgc3xybskr418cyvbx7vqs994yv9w8zf98gxvbvm";
-    url = "https://www.pureref.com/download.php";
-  };
+  src = runCommand "PureRef-${version}_x64.Appimage" {
+    buildInputs = [ curl gnugrep cacert ];
```

```suggestion
    nativeBuildInputs = [ curl gnugrep cacert ];
```
curl and grep are binaries, and cacert needs to be in `nativeBuildInputs` for the nix setup hook